kafka/generated_errors.go

```diff
@@ -1,6 +1,6 @@
 package kafka
-// Copyright 2016-2021 Confluent Inc.
-// AUTOMATICALLY GENERATED ON 2021-12-08 12:44:39.243338672 +0100 CET m=+0.000248284 USING librdkafka 1.8.2
+// Copyright 2016-2022 Confluent Inc.
```

Comment:
revert changes to this file, they'll conflict with the 1.9.0 import (or simply make this comment incorrect)